src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -277,7 +277,18 @@ void POWERMGNT::setPower(PowerLevels_e Power)
     #if defined(PLATFORM_ESP32)
     if (POWER_OUTPUT_DACWRITE)
     {
-        Radio.SetOutputPower(0b0000);
+        if (POWER_OUTPUT_VALUES2 != nullptr)
+        {
+#if defined(RADIO_SX127X)
+            Radio.SetOutputPowerRaw(POWER_OUTPUT_VALUES2[Power - MinPower]);
+#else
```

Comment:
In the spirit of commenting and helping future us, can this be a defined(RADIO_SX1280) and then an extra else with a ERRLN warning of an unsupported case?  This will help with finding all of the HW specific code for RADIO_SX12XX and throw up a warning with new transceivers,

Reply:
This is resolved in a better way by removing the `Raw` method altogether 😄